src/lang/base/GlobalConfig.ml

```diff
@@ -125,22 +125,28 @@ let stdlib_dirs = ref []
 (* List of directories to look for stdlib.
  * Entries from scilla_stdlib_env will be first. *)
 let get_stdlib_dirs () =
-  let env_dirs = 
-    match (Sys.getenv_opt scilla_stdlib_env) with 
-    | Some s when s != "" -> Str.split (Str.regexp "([;:])") s
-    | _ -> []
+  let env_dirs =
+    Option.value_map
+      (Caml.Sys.getenv_opt scilla_stdlib_env)
+      ~f:FilePath.path_of_string
+      ~default:[]
   in
     List.append env_dirs !stdlib_dirs
 
 (* Update stdlib dirs with more locations *)
 let add_stdlib_dirs dirs =
   stdlib_dirs := List.append !stdlib_dirs dirs
 
+(* File extension for Scilla contracts. *)
```

Comment:
These two lines seem to have got in during a merge conflict. The definitions are incorrect (a `.` is needed at the beginning) and are correctly defined later in the file.